apps/masterbots.ai/components/layout/profile/profile-page-sidebar.tsx

```
@@ -64,12 +71,20 @@ export const ProfileSidebar = ({ children }: any) => {
               <div className="rounded-lg">
                 <Link
                   href={`/u/${slug}/t`}
-                  onClick={() => setIsThreadsOpen(!isThreadsOpen)}
+                  onClick={handleToggleThreads}
                   className={cn(
                     "flex w-full items-center justify-between px-4 py-3",
                     "hover:bg-gray-200 dark:hover:bg-mirage transition-colors duration-200",
                     isThreadsOpen || location.pathname?.includes('/t/') ? 'bg-gray-200 dark:bg-mirage' : ''
                   )}
+                  role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)